.github/workflows/release.yml

```diff
@@ -0,0 +1,15 @@
+---
+name: sonar-release
+# This workflow is triggered when publishing a new github release
+# yamllint disable-line rule:truthy
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  release:
+    permissions:
+      id-token: write
+      contents: write
+    uses: SonarSource/gh-action_release/.github/workflows/main.yaml@v5
```

Comment:
Why doing a release/promotion to repox if we want to hand over the repo to the community?

Reply:
We can add this file, and do the release ourselves. As we will need a release to update the SDK itself.
Or we can ask DevInfra to do it manually for us.

This is cheaper/faster. In case the handover happens, this file, together with few others related to the pipeline, will be deleted.